css/at-rules.json

```
@@ -120,6 +120,17 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@counter-style"
   },
+  "@container": {
+    "syntax": "@container <container-condition># {\n  <group-rule-body>\n}",
```

I think this should be `<block-contents>`, also suggested here: https://github.com/mdn/data/pull/853#discussion_r1896859569

```suggestion
    "syntax": "@container <container-condition># {\n  <block-contents>\n}",
```